centreon/www/front_src/src/Dashboards/SingleInstancePage/Dashboard/utils.ts

```diff
@@ -1,8 +1,163 @@
-import { equals } from 'ramda';
+import {
+  T,
+  always,
+  cond,
+  equals,
+  flatten,
+  groupBy,
+  identity,
+  uniq
+} from 'ramda';
 
 export const isGenericText = equals<string | undefined>('/widgets/generictext');
 export const isRichTextEditorEmpty = (editorState: string): boolean => {
   const state = JSON.parse(editorState);
 
   return equals(state.root.children?.[0]?.children?.length, 0);
 };
+
+export const getDetailsPanelQueriers = (data): object => {
+  const uuid = data?.services?.[0].uuid;
+
+  const hostId = uuid?.split('-')[0]?.slice(1);
+  const serviceId = uuid?.split('-')[1]?.slice(1);
+
+  const resourcesDetailsEndpoint = `/centreon/api/latest/monitoring/resources/hosts/${hostId}/services/${serviceId}`;
```

Comment:
The '/centreon' might change from a platform to another. Use the `centreonBaseURL` from centreon ui